application-urlshortener-ui/src/main/resources/URLShortener/ConflictList.xml

```diff
@@ -0,0 +1,393 @@
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+-->
+
+<xwikidoc version="1.5" reference="URLShortener.ConflictList" locale="">
+  <web>URLShortener</web>
+  <name>ConflictList</name>
+  <language/>
+  <defaultLanguage/>
+  <translation>0</translation>
+  <creator>xwiki:XWiki.Admin</creator>
+  <parent>URLShortener.WebHome</parent>
+  <author>xwiki:XWiki.Admin</author>
+  <contentAuthor>xwiki:XWiki.Admin</contentAuthor>
+  <version>1.1</version>
+  <title>$services.localization.render('urlshortener.conflicts.title')</title>
+  <comment/>
+  <minorEdit>false</minorEdit>
+  <syntaxId>xwiki/2.1</syntaxId>
+  <hidden>true</hidden>
+  <content>{{warning}}
+{{translation key="urlshortener.conflicts.warning.slow"/}}
+{{/warning}}
+
+{{translation key="urlshortener.conflicts.description"/}}
+
+
+{{velocity output="false"}}
+#template('hierarchy_macros.vm')
+#macro (makeConflictsSection $map)
+  {{html}}
+  #set ($dummyString = '')
+  #set ($noPagesDisplayed = true)
+  #foreach ($pageID in $map.keySet())
+    #if ($map.get($pageID).get('size') &gt; 1)
+      #set ($noPagesDisplayed = false)
+      &lt;section&gt;
+        &lt;h2&gt;
+          $escapetool.xml($services.localization.render(
+            'urlshortener.conflicts.title.section',
+            [$pageID, $map.get($pageID).get('size')]
+          ))
+        &lt;/h2&gt;
+        &lt;table&gt;
+        #foreach ($subwiki in $map.get($pageID).keySet())
+          ## Check to skip the 'size' attribute.
+          #if ($subwiki.class != $dummyString.class)
+            #foreach ($foundDocRef in $map.get($pageID).get($subwiki))
+              &lt;tr data-old-page-id='$escapetool.xml($pageID)' data-page-ref='$escapetool.xml($foundDocRef)'&gt;
+                #if ($hasAdmin)
+                &lt;td&gt;
+                  &lt;button class='btn urlshortener-regenerate-btn'&gt;
+                    $escapetool.xml($services.localization.render('urlshortener.conflicts.regenerate.button'))
+                  &lt;/button&gt;
+                &lt;/td&gt;
+                #end
+                &lt;td&gt;
+                {{/html}}
+                #if ($map.get($pageID).get($subwiki).size() &gt; 1)
+                  $services.icon.render('exclamation')
+                #else
+                  $services.icon.render('warning')
+                #end
```

Comment:
You can use [renderHTML](https://extensions.xwiki.org/xwiki/bin/view/Extension/Icon%20Theme%20Application#HForDevelopers) so to not close the html macro